src/orion/algo/space.py

```diff
@@ -346,6 +346,13 @@ def __init__(self, name, prior, *args, **kwargs):
             raise ValueError("Lower bound {} has to be less than upper bound {}"
                              .format(self._low, self._high))
 
+        precision = kwargs.pop('precision', 4)
```

Comment:
I would add precision in __init__ signature instead and document it in the docstring.

Reply:
If I do that it breaks a bunch of tests for reasons unclear to me. And shouldn't it be with low and high?

Reply:
Oh, can you share the stacktrace of a test that fails because of this modification?

Reply:
From offline chat it seems adding other arguments before `*args` and `**kwargs` is not compatible with positional arguments for the priors. For example ``normal(0.9)`` would pass ``0.9`` as the first positional argument to ``Real()`` ( ``precision`` if we added it) and thus ``*args`` would be left empty and the prior would not receive its argument. The priority is to keep the interface simple for users, and thus support positional arguments in prior, so I would agree to use ``kwargs`` as currently done it this PR.